### DISCUSSION TITLE
Convert Python file having pickle file to exe

### DISCUSSION THREAD

**Discussion by Nee-DS** [MaartenGr/BERTopic#1311] [General]
Hi, 
This is a general question, but I have now the Bertopic pickle file and I need to create an executable .exe file from the Python file containing the .pkl file and predict functions. I am not able to convert that file by pyinstaller.
Any help will be appreciated.

Thanks

**Comment by MaartenGr** [comment]
I have not worked with `pyinstaller` before. Having said that, what is the issue that you are facing exactly? Also, is it related to BERTopic specifically or to `pyinstaller`?

**Comment by Nee-DS** [comment]
I did R&D and found that it depends on the pickle file on the functionality of pyinstaller. My aim is to get an exe file out of the Python file which has a predict function for Bertopic, and so has a pickle file. I was wondering if anyone has created an exe file from a Python file having a pickle file of the ML model. I am also asking this question in the pyinstaller forum. Any suggestions/comments here will be highly appreciated.
